mapper/src/main/kotlin/org/taktik/icure/services/external/rest/v2/mapper/embed/ContactParticipantMapper.kt

```diff
@@ -0,0 +1,13 @@
+package org.taktik.icure.services.external.rest.v2.mapper.embed
+
+import org.mapstruct.InjectionStrategy
+import org.mapstruct.Mapper
+import org.taktik.icure.entities.embed.ContactParticipant
+import org.taktik.icure.services.external.rest.v2.dto.embed.ContactParticipantDto
+
+@Mapper(componentModel = "spring", uses = [], injectionStrategy = InjectionStrategy.CONSTRUCTOR)
+interface ContactParticipantV2Mapper {
```

Comment:
We need custom code here for backward compatibility: merge from List and Map (by prioritizing List) and copy to List and Map (best-effort)